sechub-web-ui-solution/docker/nginx/nginx.conf

```diff
@@ -0,0 +1,72 @@
+worker_processes 1;
```

Comment:
Do you limit the nginx to one core on purpose? Otherwise `auto`  could be an alternative.

Reply:
@lorriborri has set this value.
I left it because I know "one process per container" as best-practise in Kubernetes.